tests/backup/backup_test.go

```diff
@@ -85,6 +85,17 @@ var _ = Describe("{UserGroupManagement}", func() {
 	})
 })
 
+// Reference test to access the cloud config details from the global variable
+var _ = Describe("{testParseConfig}", func() {
+	StartTorpedoTest("testParseConfig", "testcreds", nil, 11111)
```

Comment:
I thought this test was removed.

Reply:
it is removed .. only for testing purpose i just add it later i will remove while merging

Reply:
only for testing purpose, it is not there in master